packages/input-selection/test/RoundRobinRandomImprove.test.ts

```diff
@@ -1,6 +1,15 @@
 import { Cardano } from '@cardano-sdk/core';
 import { MockChangeAddressResolver, SelectionConstraints } from './util';
 import { TxTestUtil } from '@cardano-sdk/util-dev';
+import {
+  babbageSelectionParameters,
+  cborUtxoSetWithManyAssets,
+  getCoreUtxosFromCbor,
+  txBuilder,
+  txEvaluator
+} from './vectors';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { defaultSelectionConstraints } from '@cardano-sdk/tx-construction';
```

Comment:
Is this a circular ref? If so, probably best to hoist this somewhere. And if not possible then duplicating some functionality in tests is ok.
```suggestion
import { defaultSelectionConstraints } from '?';
```

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1380/commits/254e1af80399d849b1cab2a29e55591cf6418342